pkg/cryptoutils/publickey.go

```diff
@@ -55,6 +55,19 @@ func UnmarshalPEMToPublicKey(pemBytes []byte) (crypto.PublicKey, error) {
 	return x509.ParsePKIXPublicKey(derBytes.Bytes)
 }
 
+// UnmarshalPEMToECDSAKey converts a PEM-encoded byte slice into an *ecdsa.PublicKey.
+func UnmarshalPEMToECDSAKey(pemBytes []byte) (*ecdsa.PublicKey, error) {
```

Comment:
Do you think we should make this a private function of rekorpubs? I'd prefer to encourage package users to use the more generic PEM to crypto.PublicKey method.